scripts/artifacts-building/user_rcbops_artifacts_building.yml

```diff
@@ -31,3 +31,30 @@ openstack_repo_url: "{{ rpco_mirror_base_url }}"
 # default variant.
 lxc_image_cache_server: images.linuxcontainers.org
 
+# When we build python/container artifacts we pull the default
+# LXC cache variant from LXC upstream which may have packages
+# installed which are more recent than those we have available
+# in our artifacted apt repository. Here we figure out which
+# packages are installed, but not available in a configured
+# source, then we downgrade them to the latest available in
+# the configured sources.
+lxc_cache_prep_pre_commands: |
+    if [ -a /etc/resolv.conf ]; then
+      mv /etc/resolv.conf /etc/resolv.conf.org
+    fi
+    {% for resolver in lxc_cache_prep_dns %}
```

Comment:
It's in the role, but this var is only evaluated when the role executes so the default is used. If you feel it's better we could hard code it seeing as it's only used in CI.